hat/assets/js/apps/Iaso/domains/orgUnits/groups/components/Filters.tsx

```diff
@@ -0,0 +1,199 @@
+        };
+    });
+};
+
+const Filters: FunctionComponent<Props> = ({ params }) => {
+    const { filters, handleSearch, handleChange, filtersUpdated } =
+        useFilterState({ baseUrl, params, withPagination: true });
+
+    const [textSearchError, setTextSearchError] = useState<boolean>(false);
+
+    const { data: projects, isFetching: isFetchingProjects } =
+        useGetProjectsDropdownOptions();
+    const { data: dataSources, isLoading: areSourcesLoading } =
+        useGetDataSources({ ...params, ...filters });
+
+    const { data: sourceVersions, isLoading: areSourceVersionsLoading } =
+        useDataSourceVersions();
+
+    const [projectIds, setProjectIds] = useState<string | undefined>(
+        filters?.project_ids,
+    );
+    const [dataSource, setDataSource] = useState<string | undefined>(
+        filters?.dataSource,
+    );
+    const [version, setVersion] = useState<string | undefined>(
+        filters?.version,
+    );
+
+    const dataSourceDropDown = useMemo(
+        () => dataSourceByProjectId(dataSources?.sources, filters?.project_ids),
+        [dataSources, filters?.project_ids],
+    );
+
+    const sourceVersionsDropDown = useMemo(
+        () => versionsBySource(sourceVersions, filters?.dataSource),
+        [filters?.dataSource, sourceVersions],
+    );
+
+    const handleChangeSelect = useCallback(
+        (key, newValue) => {
+            let value = newValue;
+
+            if (newValue === null) {
+                value = undefined;
+            }
+            if (key === 'project_ids') {
+                filters.dataSource = undefined;
+                filters.version = undefined;
+                setProjectIds(value);
+                setDataSource(undefined);
+                setVersion(undefined);
+            }
+            if (key === 'dataSource') {
+                filters.version = undefined;
+                setVersion(undefined);
+                setDataSource(value);
+            } else if (key === 'version') {
+                setVersion(value);
+            }
+            handleChange(key, value);
+        },
+        [filters, handleChange, setProjectIds, setDataSource, setVersion],
+    );
+    const { formatMessage } = useSafeIntl();
+
+    return (
+        <Grid container spacing={2}>
+            <Grid item xs={12} sm={6} md={3}>
+                <InputComponent
+                    keyValue="search"
+                    onChange={handleChange}
+                    value={filters.search}
+                    type="search"
+                    label={MESSAGES.search}
+                    onEnterPressed={handleSearch}
+                    blockForbiddenChars
+                    onErrorChange={setTextSearchError}
+                />
+            </Grid>
+            <Grid item xs={12} md={3}>
+                <InputComponent
+                    keyValue="project_ids"
+                    onChange={handleChangeSelect}
+                    value={projectIds}
```

Comment:
I think this projectIds should be renamed into projectId because the select component isn't multi-select.